src/com/cdd/bao/editor/DetailPane.java

```diff
@@ -335,19 +329,43 @@ public void actionValueMove(int dir)
 
 		valueList.get(newIndex).fieldURI.requestFocus();
 	}
-	public void actionLookupURI()
+
+	@Override
+	public void actionLookupURI(int focusIndex)
 	{
-		if (focusIndex < 0) return;
+		System.out.println("focusIndex=" + focusIndex + "; valueList.length=" + valueList.size());
 		ValueWidgets vw = valueList.get(focusIndex);
 		String uri = ModelSchema.expandPrefix(vw.fieldURI.getText());
-		if (uri.length() == 0) {actionLookupName(); return;}
-	
+		if (uri.length() == 0)
```

Comment:
This isn't wrong, but multiple statements on the same line are allowed; by default leave it (i.e. roll it back).